idb/postgres/postgres_migrations.go

```diff
@@ -821,6 +821,28 @@ func m7RewardsAndDatesPart2UpdateAccounts(db *IndexerDb, accountData []addressAc
 	return nil
 }
 
+func warnUser(db *IndexerDb, maxRound uint32) error {
+	query := "SELECT COUNT(*) FROM account WHERE (created_at IS NULL) OR (created_at <= $1)"
+	row := db.db.QueryRow(query, maxRound)
+
+	var count uint64
+	err := row.Scan(&count)
+	if err != nil {
+		return fmt.Errorf("m7: unable to query the number of rows: %v", err)
+	}
+
+	if count > 10000000/3*4 {
+		db.log.Print("the migration m7 is likely to use more than 4GB of RAM")
```

Comment:
We should probably make this comment more explanatory - "The current migration (m7) is likely to use more than 4GB of RAM. The avoid overuse, to process stopped automatically"